<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-BS7L7CT0" failures="1" tests="4" name="PetStoreTest" time="10.002" errors="0" timestamp="2025-03-15T14:27:13 IST">
  <testcase classname="api.tests.UserTests2" name="createUser" time="3.609"/>
  <testcase classname="api.tests.UserTests2" name="getUser" time="1.213">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.tests.UserTests2.getUser(UserTests2.java:62)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- getUser -->
  <testcase classname="api.tests.UserTests2" name="updateUser" time="2.639"/>
  <testcase classname="api.tests.UserTests2" name="deleteUser" time="1.223"/>
</testsuite> <!-- PetStoreTest -->
